front/app/modules/commercial/custom_idea_statuses/admin/containers/index.tsx

```diff
@@ -145,31 +154,30 @@ const IdeaStatuses = () => {
             <T value={defaultStatus.attributes.title_multiloc} />
           </FlexTextCell>
           <Buttons>
-            <Tooltip
-              placement="top"
-              theme="light"
-              disabled={false}
-              content={
-                <FormattedMessage
-                  {...messages.defaultStatusDeleteButtonTooltip}
-                />
+            {/* This DeleteStatusButton is a dummy button. The default status can never be deleted, 
```

Comment:
Makes sense! We could put a locked icon in there when it's the default behaviour? Not important, though.

Reply:
We already do! :) 
<img width="715" alt="image" src="https://github.com/CitizenLabDotCo/citizenlab/assets/16427929/215048d7-46ea-4eb0-bb66-28fd77332302">